syntax = "proto3";

service Status{
    rpc status(status_request) returns (status_response){}
}

message status_request{
    int64 soldier_id = 1;
}

message status_response{
    bool alive = 1;
}